lib/service_actor/checks/must_check.rb

```diff
@@ -53,11 +53,17 @@ def initialize(input_key:, actor:, nested_checks:, value:)
     @value = value
   end
 
-  def check
+  def check # rubocop:disable Metrics/MethodLength
     @nested_checks.each do |nested_check_name, nested_check_conditions|
       check, message = define_check_and_message_from(nested_check_conditions)
 
-      next if check.call(@value)
+      begin
+        next if check.call(@value)
+      rescue StandardError => e
+        message =
+          "The \"#{@input_key}\" input on \"#{@actor}\" has an error " \
+            "in the code inside \"#{nested_check_name}\": #{e}"
+      end
```

Comment:
Perhaps instead of disabling the `Metrics/MethodLength` cop we can extract the call in a method. E.g.:

```
message = call_check(nested_check_name, nested_check_conditions)
next unless message
```

With:

```
# …
def call_check(check_name, check_condition)
  check, message = define_check_and_message_from(check_condition)
  return if check.call(value)
  message
rescue StandardError => e
  "The \"#{input_key}\" input on \"#{actor}\" has an error " \
  "in the code inside \"#{check_name}\": #{e.class} #{e}"
end
```

Reply:
I agree. I'll do it soon.

Reply:
Done.

Reply:
Haha: https://github.com/sunny/actor/pull/105/commits/13381d6d3f9023293a4ae1bd51c1ffca4a3a3ead